package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "description": "<a name=\"readme-top\"></a>",
-  "main": "index.js",
+  "main": "webpack.config.js",
   "scripts": {
     "dev": "webpack --mode development",
```

Comment:
- [ ] [OPTIONAL] I would highly suggest you to configure your package.json for opening the project in the server. You can do this by adding `"start": "webpack serve --open"` into script instead of `"start": "webpack --config webpack.config.js --watch"`